common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -40,6 +40,7 @@
   "command.wynntils.lootrun.undoStandNear": "Stand near a previous spot to undo movements.",
   "command.wynntils.lootrun.undoSuccessful": "Successfully undid movements.",
   "command.wynntils.map.description": "Manage map related settings",
+  "command.wynntils.menu.description": "Opens the Wynntils Menu",
   "command.wynntils.quest.description": "List, show and track quests",
   "command.wynntils.reauth.tryReauth": "Disconnecting from Hades, and trying to reauthenticate...",
   "command.wynntils.reloadCaches.reloading": "Reloading caches...",
```

Comment:
Actually, no, I don't like this. Move this command into the wynntils command as "wynntils menu"

Reply:
I have no idea why this comment is here though, thanks Github Mobile.

Reply:
I tried, but it doesn't work for some reason. Should I commit so you can take a look? Basically suggestion and help command shows it but when I try to execute it, I am getting an error about unknown command.

Reply:
Sure

Reply:
Alright it was very small thing that I missed